### PR TITLE
DX-1714: Add svgs for dealer storefront cms contemporary changes

### PR DESCRIPTION
**Pull request by cormickjbrowne**

### CODE REVIEW

File: src/image-upload-drag.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-do-not-edit" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (hellatan) [inline src/image-upload-drag.svg]: Class name is wrong

File: src/download.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg version="1.1" class="dibsvg-svg dibsvg-download" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (hellatan) [inline src/download.svg]: This is replacing a font awesome icon?

Comment (cormickjbrowne) [inline src/download.svg]: Yeah right now we use fa icons on the image cropper elements and we're replacing them with buttons with svgs/text.

**Comment by cormickjbrowne** [comment]
Blocks: https://github.com/1stdibs/app-dealer-tools/pull/292

**Comment by cormickjbrowne** [comment]
@jrdrg did you publish this? What's the new version?

**Comment by jrdrg** [comment]
Sorry, I didn't before, just did now

+ dibs-vg@1.27.0

**Pull request merged**